### PR TITLE
3796 undo and retry

### PR DESCRIPTION
**Pull request by sergiomilici**

### CODE REVIEW

File: www/js/services/transaction_service.js [hunk 1]
```diff
@@ -72,5 +76,34 @@ app.service('TransactionService', function($q, UserService, RequestParameterBuil
...
+        .setField('created', transaction.created)
+        .setField('force', -1)
+        .setField('goods', transaction.goods)
+        .getParams();
+
+    return this.makeRequest_(params, sellerAccountInfo)
+      .then(function(res) {
+        return res.data;
+      })
+      .then(function(transactionResult) {
```
Comment (jpdimond) [inline www/js/services/transaction_service.js:97]: The user needs some way of knowing that this has been successful. Could you use the same success/error screen with the updated message? e.g. "The transaction of $0.25 was successfully reversed."

File: www/js/controllers/transaction_result_controller.js [hunk 2]
```diff
@@ -26,7 +26,18 @@ app.controller('TransactionResultCtrl', function($scope, $state,
...
-    customerName: $scope.customer.name,
-  }
+    customerName: $scope.customer.name
+  };
+
+  $scope.undoTransaction = function() {
+    $ionicLoading.show();
+    TransactionService.undoTransaction(TransactionService.lastTransaction)
+      .then(function(transactionResult) {
+        console.log("Undo transaction: ", transactionResult);
```
Comment (jpdimond) [inline www/js/controllers/transaction_result_controller.js]: should this console log still be here?

File: www/js/services/transaction_service.js (hunk 1 above)
Comment (sergiomilici) [inline www/js/services/transaction_service.js:97]: ok. Changing it.

File: www/js/controllers/transaction_result_controller.js (hunk 2 above)
Comment (sergiomilici) [inline www/js/controllers/transaction_result_controller.js]: no.

**Pull request merged**